app.js

```diff
@@ -1,3 +1,63 @@
+const projects = [
```

Comment:
- [ ] I found some differences between our project pop-up and Figma. Let's check the screenshots below:

| Figma | Yours |
| --- | --- |
| ![Screenshot from 2022-11-16 04-07-41](https://user-images.githubusercontent.com/63087491/202074698-0102c68e-19f6-444e-96de-fb13bc0ee431.png) | ![Screenshot from 2022-11-16 04-04-59](https://user-images.githubusercontent.com/63087491/202074958-85e2f871-78d0-414b-8c08-5875f18518a7.png) |


- Based on the requirements of this project, the popup should follow the design in Figma(colors, font, images, text, margins). As we can all see on the above screenshots, the description of your pop is missing on desktop size, and could only be seen on the mobile screen(please also make sure to include text equivalent to that of Figma ). The value of the attribute` name` is not aligned as in Figma.